### DISCUSSION TITLE
extra amp required?

### DISCUSSION THREAD

**Discussion by marc-portier** [micro-nova/AmpliPi#193] [General] [answered]
just to be clear on the supported stereo audio output

will I need an extra amplifier before actually driving my speakers? (or put differently- does this assume active speakers?)

**Comment by Lohrer** [comment, marked as answer]
No extra amplifier is required! Each of the 6 stereo zone outputs are powered by amplifiers. Check out the hardware section of https://www.micro-nova.com/amplipi for more information.

There are also line-level RCA outputs, but those are in addition to the amplified outputs.
